src/sparseml/export/export.py

```diff
@@ -109,14 +113,17 @@ def export(
     :param task: Optional task to use for exporting the model.
         Defaults to None.
     """
+    if task is not None:
+        task = task.replace("_", "-").replace(" ", "-")
+    if integration is not None:
+        integration = integration.replace("_", "-").replace(" ", "-")
```

Comment:
nit: helper fn?

Reply:
+1, this should ideally be resolved at the registry mixin level (all lower case, remove special characters and whitespace)

Reply:
Please take a look at: https://github.com/neuralmagic/sparsezoo/pull/404
It's been sitting for a while in the "purgatory", but has already been reviewed once by Sara.
@Satrat , @bfineran if you approved this sparsezoo feature, we could remove the need for unifiying the `integration` string (`RegistryMixin` will take care of it). For the task itself, I have also a good solution, will implement in a second.